### ISSUE TITLE
[FEATURE] Friendlier error message when video access is restricted by subscription level

### ISSUE THREAD

**Issue by b-mck** [robvdpol/RaceControl#135]
**Is your feature request related to a problem? Please describe.**
Currently when you attempt to access a video that's not available yet because you have the lower-level subscription, it simply fires a popup stating "An error occurred retrieving the stream URL."

**Describe the solution you'd like**
An error (popup or otherwise) that notes the video is not available because the 48-hour wait period has not elapsed yet would be preferable.

**Describe alternatives you've considered**
Hiding the videos from the listing based on availability of stream would also work.

**Additional context**
Currently there's no indicator of whether you're subscribed to the full, live-streaming level of content or only the race replays.

**Comment by b-mck** [comment]
>  A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]

**Comment by b-mck** [comment]
Apologies, got tripped up by the UI of Github Issues and accidentally closed.

**Comment by AlwinCrasto** [comment]
Yes a descriptive message would be really great for those of us on F1 TV Access. As detailed on [this page](https://www.formula1.com/en/toolbar/content_schedule.html) there could be a host of different reasons why a video is not playable.
- Some videos (mainly session replays of the past 2 years and press conference videos) are simply not viewable in some countries and are playable only after a 2 year delay.
- The session replay video may be available in my country but only 2 days after the event.
- Some session replay videos (F2, F3 and Porsche Supercup replays) may never become available at all in some countries.
- Some archive content like the the "Race to Perfection" documentary series is not available in some regions, including mine, for broadcasting rights reasons or whatever.

I'm not sure if the API can be used to determine one of the above reasons of a video not being playable but perhaps the developer can comment on the feasibility of it. 

On a related note, can anything be done about using Race Control with a VPN to let those of us not having the Pro tier in our countries unlock at least the race replays and the complete archive content? At present Race Control won't even log me in to my account when I VPN to another country.
